tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -448,29 +449,6 @@ _handle_ingress_aks() {
 	popd
 }
 
-# Implements the ingress handler for servernode
-# this is useful on kcli or anywhere where cluster IPs are accessible
-# from the testing machines.
-#
 _handle_ingress_nodeport() {
-	pushd "${COCO_KBS_DIR}/config/kubernetes/overlays"
-
-	cat > nodeport_service.yaml <<EOF
-# Service to expose the KBS on nodes
-apiVersion: v1
-kind: Service
-metadata:
-  name: kbs-nodeport
-  namespace: "$KBS_NS"
-spec:
-  selector:
-    app: kbs
-  ports:
-  - protocol: TCP
-    port: 8080
-    targetPort: 8080
-  type: NodePort
-EOF
-	kustomize edit add resource nodeport_service.yaml
-	popd
+	export DEPLOYMENT_DIR=nodeport
```

Comment:
And here add a comment line explaining that by exporting DEPLOYMENT_DIR=nodeport the kbs deploy script install the nodeport service.

Reply:
@wainersm changes applied